python/src/tests/data/cell_set_types.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
this file contains different flavours of cell sets, used in various tests for return data of s3 download calls. The data is a replacement of `cell_set_responses` in `python/src/worker/helpers/mock_s3.py`